assets/src/edit-story/components/inspector/prepublish/styles.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Headline,
+  Icons,
+  Text,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
+import { MAX_NUMBER_FOR_BADGE } from './constants';
+
+function annotateNumber(number) {
+  if (number <= MAX_NUMBER_FOR_BADGE) {
+    return number;
+  }
+  return `${MAX_NUMBER_FOR_BADGE}+`;
+}
+
+export const NumberBadge = styled.span`
+  display: inline-flex;
+  height: 20px;
+  width: 20px;
+  line-height: 20px;
+  justify-content: center;
+  border-radius: 50%;
+  font-size: ${({ number }) =>
+    number > MAX_NUMBER_FOR_BADGE ? '10px' : '12px'};
+  &::after {
+    content: ${({ number }) => `"${annotateNumber(number)}"`};
+  }
+  color: ${({ theme }) => theme.colors.bg.secondary};
+  background-color: ${({ isRecommended, theme }) =>
+    isRecommended ? theme.colors.fg.linkNormal : theme.colors.fg.negative};
+`;
+NumberBadge.propTypes = {
+  isRecommended: PropTypes.bool,
+  number: PropTypes.number,
+};
+
+export const TitleWrapper = styled.span`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+export const PanelTitle = styled(Headline).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XX_SMALL,
+  as: 'span',
```

Comment:
This title exists inside of an h2 element on the panel title, having a nested heading is redundant.

Reply:
💭 should we just apply the styles we want directly then? 
```js
export const PanelTitle = styled(Text).attrs({
  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XX_SMALL,
  forwardedAs: 'span',
})`
    ${themeHelpers.expandPresetStyles(...)
    ...
```

I think the way you did it is more readable though so let's keep it how it is?

Reply:
🤔  i'm _sure_ there's a way to clean up the titles a bit. the prepublish titles threw a wrench in the old title structure, i think getting some of these PRs merged will clear the way for cleaning up the titles and handling the different use cases, this is a hard moving target to hit.